libpod/container.go

```diff
@@ -236,6 +236,8 @@ type ContainerOverlayVolume struct {
 	Dest string `json:"dest"`
 	// Source specifies the source path of the mount.
 	Source string `json:"source,omitempty"`
+	// Chown changes the source UID and GID ownership.
+	Chown bool `json:"chown,omitempty"`
```

Comment:
Can you make this a more generic `Options []string` instead? It seems like we will have more of these later on, and the other volume types don't have per-option bools.

Reply:
done